app/ledger.js

```diff
@@ -981,8 +986,12 @@ var stickyP = (publisher) => {
   var result = (siteSetting) && (siteSetting.get('ledgerPayments'))
 
   // NB: legacy clean-up
-  if ((typeof result === 'undefined') && (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined')) {
-    result = synopsis.publishers[publisher].options.stickyP
+  if (typeof result === 'undefined') {
+    if (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined') {
+      result = synopsis.publishers[publisher].options.stickyP
+    } else {
+      result = true
+    }
     appActions.changeSiteSetting(pattern, 'ledgerPayments', result)
   }
   delete synopsis.publishers[publisher].options.stickyP
```

Comment:
Ok, but why would be failing now. It was outside the if statement before. It was strange for me, but I left it there because it was there before. Will move it inside the if